build/derby-bootstrap.sh

```diff
@@ -26,13 +26,13 @@ java -jar ${WORKSPACE}/fhir-persistence-schema/target/fhir-persistence-schema-*-
   --update-schema
 java -jar ${WORKSPACE}/fhir-persistence-schema/target/fhir-persistence-schema-*-cli.jar \
   --db-type derby --prop db.database=${DB_LOC}/profile --prop db.create=Y \
-  --prop resourceTypes=Patient,Group,Practitioner,PractitionerRole,Person,RelatedPerson,Organization,Location,Observation,MedicationAdministration,StructureDefinition,ElementDefinition,CodeSystem,ValueSet,Encounter,Condition,MedicationRequest,Coverage,ServiceRequest,CarePlan,CareTeam,Claim,DiagnosticReport,ExplanationOfBenefit,Immunization,Procedure,Medication,Provenance,Consent \
+  --prop resourceTypes=Patient,Group,Practitioner,PractitionerRole,Person,RelatedPerson,Organization,Location,Observation,MedicationAdministration,StructureDefinition,ElementDefinition,CompartmentDefinition,CodeSystem,ValueSet,Encounter,Condition,MedicationRequest,Coverage,ServiceRequest,CarePlan,CareTeam,Claim,DiagnosticReport,ExplanationOfBenefit,Immunization,Procedure,Medication,Provenance,Consent \
```

Comment:
Do you think it's worth factoring out the common set of resource types into a variable?

Reply:
I also wonder if we should allow the resource types list to come from a file, which we could then use in multiple places. Minor.

Reply:
yes.  everytime i touch these I think that.  i've just never made it a priority...

Reply:
ok, i tried my hand at this.  now we have build/common/set_tenant1_datastore_vars.sh which exports the variables and I call that from each workflow instead of repeating the values each time

the one exception to this is our windows build which is using powershell...i didn't bother modifying it there.  instead, i just left a note in the set_tenant1_datastore_vars.sh script that says you should update the .ps1 script with each edit